WORKSPACE.bazel

```diff
@@ -7,7 +7,10 @@ local_repository(
 
 load("//cuda:repositories.bzl", "register_detected_cuda_toolchains", "rules_cuda_dependencies")
 
-rules_cuda_dependencies()
+rules_cuda_dependencies(
+    cuda_version = "11.8",
+    dll_runtime_system_provided = False,
+)
```

Comment:
Don't change this macro, it is for other dependent rules.
And please don't introduce `cuda_version` or any kind of version specific config into workspace rule of macro. It is not configurable.